source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailView.tsx

```diff
@@ -286,11 +286,14 @@ export const CompensationRequisitionDetailView: React.FunctionComponent<
             <SectionField label="Expropriation vesting date" labelWidth="4">
               {prettyFormatDate(compensation.expropriationVestingDate)}
             </SectionField>
-            <SectionField
+            {/* TODO : Remove */}
```

Comment:
Is this part of this PR?

Reply:
yes, so my test passes because no one worked on the removal of this field.